proposals/0118-partitioned-epoch-reward-distribution.md

```diff
@@ -0,0 +1,265 @@
+---
+simd: '0118'
+title: Partitioned Epoch Rewards Distribution
+authors: Haoran Yi (Anza), Justin Starry (Anza), Tyera Eulberg (Anza)
+category: Standard
+type: Core
+status: Draft
+created: 2024-02-16
+feature: (fill in with feature tracking issues once accepted)
+supersedes: '0015'
```

Comment:
Certainly something. I like `superceded-by`... will see if that's acceptable to Jacob

Reply:
I also update the simd-0015 header to status: Accepted, because that's technically what it was before this superceding simd. I'm not totally clear on what the status-change process should look like, though (cc @jacobcreech )

Reply:
That sounds correct. All of these SIMDs were technically "Accepted" and I'll need to update them. Let me add `superceded-by` so that the CI will play nice. This also helps a lot when we eventually get a website to display SIMDs.

Reply:
Added in https://github.com/solana-foundation/solana-improvement-documents/pull/126